int64 request #0: return nbv point
---
geometry_msgs/Pose nbv_pose
